lib/SIL/SILBasicBlock.cpp

```diff
@@ -336,6 +337,79 @@ bool SILBasicBlock::isEntry() const {
   return this == &*getParent()->begin();
 }
 
+//===----------------------------------------------------------------------===//
+//                            Leaking BB Analysis
```

Comment:
Can you change all of the references here to ARC/Leaking BBs/etc to refer to trapping block analysis/etc. I'll show you the ones that I see.